packages/bus-sqs/src/sqs-transport.ts

```diff
@@ -224,7 +269,7 @@ export class SqsTransport implements Transport<SQS.Message> {
       }
     )
 
-    const serviceQueueAttributes: QueueAttributeMap = {
+    const serviceQueueAttributes: Record<string, string> = {
```

Comment:
is it possible to align this with the target type? something like:

```suggestion
    const serviceQueueAttributes: CreateQueueCommandInput['Attributes'] = {
```